src/SFA.DAS.ApprenticeCommitments.Jobs/SendInvitationRemindersRequest.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+
+namespace SFA.DAS.ApprenticeCommitments.Jobs
+{
+    public class SendInvitationRemindersRequest
+    {
+        public DateTime SendNow { set; get; }
+        public int RemindAfterDays { set; get; }
+    }
```

Comment:
I'm a little confused by this API.  What is the purpose of `SendNow`?  Is it supposed to tell the receiving API "send reminders to any apprenticeships that are incomplete after `SendNow + RemindAfterDays days`?

I think it may be clearer if the API simply received the DateTime cutoff - i.e. the instant which all apprenticeships that were created before should be reminded (if not completed).  `DateTime RemindApprenticeshipsCreatedBefore` or something analgous.  

This removes coupling between the jobs project and the API, as only the jobs project need know how to calculate the timestamp - based on its config and the time the job runs; the API only compares the given timestamp to the `CreatedAt` database row, and does not know anything about how to calculate that timestamp.

Reply:
Agree, The idea was that the logic should reside in the inner API. And this Time value could be used to stamp the registration record with this value. Therefore we could easily see which reminders were sent together

But it just adds a little confusion.